test/login.spec.js

```diff
@@ -44,21 +41,18 @@ describe('User login authentication: ', () => {
     before(async () => {
       await updateVerifiedStatus(validLoginUser.email);
     });
-    it('should return a success message with status 200', (done) => {
-      chai
+    it('should return a success message with status 200', async () => {
```

Comment:
@chingsley I still can't see any tests for the super admin user

Reply:
You are right. I am working it now...